robot_skills/src/robot_skills/torso.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, robot_name, tf_listener):
                                                               control_msgs.msg.FollowJointTrajectoryAction)
 
         # Init joint measurement subscriber
-        self.torso_sub = rospy.Subscriber('/' + self.robot_name + '/sergio/torso/measurements', JointState,
+        self.torso_sub = self.create_subscriber('/' + self.robot_name + '/sergio/torso/measurements', JointState,
```

Comment:
Indentation

Reply:
It's either going over the max line width or having these weird breaks.